src/css/profile/mobile/common/dropdownmenu.less

```diff
@@ -194,6 +191,11 @@
 		border-top: 1 * @px_base solid @color_dropdownmenu_list_group_stroke;
 		border-bottom: 1 * @px_base solid @color_dropdownmenu_list_group_stroke;
 	}
+	// top, bottom classes just describes direction in which menu is shown.
```

Comment:
you don't need define empty styles.

Reply:
Right but they are still needed.
Those classes (top or bottom) are applied by DropdownMenu.js and are used by css to animate showing menu in specified direction, for example:
```
        &.ui-dropdownmenu-options-opening.ui-dropdownmenu-options-top .ui-dropdownmenu-options {
                .animation(open-to-top 300ms);
        }
```